app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,38 @@
+<h2>Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :email %><br />
```

Comment:
- [ ] I'm unable to `signup` and `login`, When I try nothing happens on the screen I still see the same pages. As requested by the previous reviewer, Please fix the authentication issue and users should `signup` and `login` without any issues.

Reply:
@thecodechaser everything was working on my end, I made changes as requested but I ended up breaking all the code